api/managementpb/checks.proto

```diff
@@ -77,8 +110,54 @@ message ChangeSecurityChecksRequest {
 
 message ChangeSecurityChecksResponse {}
 
+message ListFailedServicesRequest {}
+
+message ListFailedServicesResponse {
+  repeated CheckResultSummary result = 1;
+}
+
+message GetFailedChecksRequest {
+  string service_id = 1;
+  PageParams page_params = 2;
+}
+
+message GetFailedChecksResponse {
+  repeated CheckResult results = 1;
+  PageTotals page_totals = 2;
+}
+
+message ToggleCheckAlertRequest {
+  // Alert ID of the check result.
+  string alert_id = 1;
+  // Silence the check result if set to true, unsilence otherwise.
+  bool silence = 2;
+}
+
+message ToggleCheckAlertResponse {}
+
 // SecurityChecks service provides public Management API methods for Security Checks Service.
 service SecurityChecks {
+  // ListFailedServices returns a list of services with failed checks.
+  rpc ListFailedServices(ListFailedServicesRequest) returns (ListFailedServicesResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/SecurityChecks/ListFailedServices"
+      body: "*"
+    };
+  }
+  // GetFailedChecks returns the checks result for a given service.
+  rpc GetFailedChecks(GetFailedChecksRequest) returns (GetFailedChecksResponse) {
```

Comment:
I think this can be replaced by using `GetSecurityCheckResults` with two filters, what do you think?

Reply:
A separate filter to return the failed checks per service, right?

I think the plan is to deprecate and eventually remove `GetSecurityCheckResults` since it's not used (at least by FE).